src/Sarif/ToDotNet/ToDotNet.targets

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <NuGetPath>$(MSBuildProjectDirectory)\..\..\.nuget</NuGetPath>
     <JsonSchemaToDotNetPath>..\packages\Microsoft.Json.Schema.ToDotNet.0.36.0\tools\JsonSchemaToDotNet.exe</JsonSchemaToDotNetPath>
```

Comment:
Note that even though I upgraded Json.Schema to 0.37.0, I left Json.Schema.ToDotNet at 0.36.0 (I didn't bother to republish it). ToDotNet actually does still depend on .NET 4.5.2 (because of Roslyn), but we don't have a build-time dependency on ToDotNet; we just run the command line tool to do the code generation.